src/device.coffee

```diff
@@ -186,7 +186,8 @@ exports.getOSVersion = ->
 		for line in lines
 			[ key, val ] = line.split('=')
 			releaseItems[_.trim(key)] = _.trim(val)
-		return releaseItems['PRETTY_NAME']
+		# Remove enclosing quotes: http://stackoverflow.com/a/19156197/2549019
+		return releaseItems['PRETTY_NAME'].replace(/^"(.+(?="$))"$/, '$1')
```

Comment:
I've only seen two cases: we either have quotes at start and end that we want remove, or we don't have quotes at all. I'm just using this to protect against cases of the string having quotes elsewhere, which we wouldn't want to remove. Especially since users can do their own OS builds.